src/Bitmap.cpp

```diff
@@ -303,10 +286,11 @@ void InitBMHead()
 	char buf[1024+3];
 	sprintf(buf, "%s.ge", OutFileBase);
 #ifdef _WIN32
```

Comment:
Can we split this change out into a `create_directory()` function with platform-dependent implementations.  I don't like the `if` statement being platform specific.

Reply:
I think I didn't intend it originally for this PR. I reverted it here so I'll probably just make a new PR for that change.